src/peft/tuners/osf/__init__.py

```diff
@@ -0,0 +1,13 @@
+from peft.utils import register_peft_method
+
+from .config import OSFConfig
+from .model import OSFModel
+
+__all__ = ["OSFConfig", "OSFModel"]
+
+register_peft_method(
+    name="osf",
```

Comment:
You might also need to register your method as a new PEFT type in `peft.utils.peft_types.PeftType`, otherwise this won't work